README.md

```diff
@@ -68,8 +68,8 @@ Website that will be used as my portifolio.
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ] [OPTIONAL] I highly suggest removing sentences that start with `>` from the README. They're meant to be guides on what to do there. 🧐️

**Examples**
![Screenshot from 2023-02-16 15-37-29](https://user-images.githubusercontent.com/17243325/219367329-3b45aad4-e99a-402d-bd7d-246236956582.png)
![Screenshot from 2023-02-16 15-37-44](https://user-images.githubusercontent.com/17243325/219367334-f606297b-1cb9-4cf7-a461-76b547f2d248.png)